src/app/proposals/LatestProposalsTable.tsx

```diff
@@ -105,7 +106,18 @@ const LatestProposalsTable = ({ proposals }: LatestProposalsTableProps) => {
       sortingFn: (rowA, rowB) => {
         const getDominantVoteType = (votesData: typeof rowA.original.votes) => {
           const { againstVotes, forVotes, abstainVotes } = votesData
-          const maxCount = Math.max(againstVotes, forVotes, abstainVotes)
+          // @TODO if used elsewhere - extract to utils
```

Comment:
Even though we don't use this logic anywhere else, it would be nice to extract it to utils for better readability.